styles.css

```diff
@@ -170,6 +182,116 @@ h2 > a:hover {
   margin-top: -0.8rem;
 }
 
+/* --- DOM Manipulation --- */
+
+.popup {
+  position: fixed;
+  visibility: hidden;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  height: 100%;
+  z-index: 3;
+  background-color: #fff;
+  margin-bottom: 2rem;
+  border-radius: 2px;
```

Comment:
- [ ] The pop-up when I close it and I open it again I can't see it. see the screenshot 👇

![kkk](https://user-images.githubusercontent.com/61562626/132542190-04d3a761-6c8f-4ebe-b660-3fe61eb6d550.gif)